test/e2e_new/data/infrastructure-aws/cluster-template.yaml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
Is there a need for a separate template here? it might be nice if we could re-use the existing template instead of having to maintain a separate template for the e2e tests.